devops/Makefile

```diff
@@ -291,6 +291,12 @@ init-redis-cluster:
 	REDIS_NODES=$$(oc get pods -n $(NAMESPACE) -l app=redis -o jsonpath='{range .items[*]}{.status.podIP}:6379 ' | sed 's/ :6379 $$/ /'); \
 	oc exec -n $(NAMESPACE) -it redis-0 -- redis-cli -a $$REDIS_PASSWORD --cluster create $$REDIS_NODES --cluster-replicas 1
 
+# Command to tell one Redis node to connect to another node and join the cluster. 
+# This is typically done when you are setting up a Redis Cluster or adding new nodes to an existing cluster to ensure they are aware of each other and can communicate effectively.
+redis-cluster-meet:
+	REDIS_PASSWORD=$$(oc get secret -n $(NAMESPACE) redis-creds -o jsonpath='{.data.password}' | base64 -d); \
+  oc get pods -n $(NAMESPACE) -l app=redis -o jsonpath='{range.items[*]}{.status.podIP} 6379 {end}' | xargs -n2 oc exec -n $(NAMESPACE) -it redis-0 -- redis-cli -a $$REDIS_PASSWORD -c CLUSTER MEET
```

Comment:
Should there be a colon before the port number? jsonpath='{range.items[*]}{.status.podIP}:6379

Reply:
I guess the command takes the variables as IP 6379, in the previous cluster create command we used ip:6379 as the command requires that. So i believe in this command this is not required. If we supply with the : , we get the below error.
ERR Invalid TCP base port specified: <ip>:6379